Assignment/Program.cs

```diff
@@ -1,7 +1,17 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter number to be checked :");
+   if (int.TryParse (Console.ReadLine (), out int num)) {
+      int count = 0;
+      for (int i = 2; i < num; i++) {
+         if (num % i != 0)
+            count++;
+         else {
+            Console.Write ($"{num} is not a prime number");
+            break;
+         }
+         if (count == num - 2)
+            Console.Write ($"{num} is a prime number");
       }
-   }
+      break;

```

Comment:
Why do you need this break here?

Reply:
I'm having a loop to handle invalid input
Example: T20 or $20
The loop will throw a message as an invalid number and this loop will continue until the user gives valid number
If valid input is validated then the loop will break